lib/oli/delivery/sections/browse.ex

```diff
@@ -98,9 +98,19 @@ defmodule Oli.Delivery.Sections.Browse do
         },
         group_by: [e.section_id]
 
+    student_enrollments =
+      from e in Enrollment,
+        join: u in User,
+        on: e.user_id == u.id,
+        where: u.can_create_sections == false,
+        select: %{
+          id: e.id,
+          section_id: e.section_id
+        }
```

Comment:
Looking at the query I think that maybe instead of querying for the `can_create_sections` field to find out if a user is a Student or an Instructor, you could add a join in this query with the `enrollments_context_roles` table, which contains the information of what type of enrollment the user has. I comment it just to think about it...

Reply:
I agree with Santi. A user who can create sections does not necessarily has to have an instructor role in all sections. A user can have the ability to create sections and be enrolled as a student in many courses. 
So, I think the right way to address the ticket is the way Santi suggests